lti_consumer/lti_1p3/consumer.py

```diff
@@ -540,18 +555,57 @@ def enable_deep_linking(
         """
         self.dl = LtiDeepLinking(deep_linking_launch_url, deep_linking_return_url)
 
+    def enable_proctoring(
```

Comment:
I am wondering if we can create a new class called `LtiProctoringConsumer` instead of enhancing the existing LtiAdvantageConsumer. This way, we can write our Proctoring logic in the new inherited class?

Reply:
:+1: for splitting this into a separate module - proctoring doesn't have the same use cases as `LtiAdvantage` in the platform now.